catalog/ui/search-ui/standard/src/main/webapp/js/view/ingest/IngestModal.view.js

```diff
@@ -51,13 +51,12 @@ define([
                 var view = this;
                 view.$('[data-toggle="upload-popover"]').popover();
                 view.fileUploadListRegion.show(new UploadList({collection: view.collection}));
-                $.ajaxSetup({
-                    headers: {directive: "STORE_AND_PROCESS"}
-                });
+
                 this.$('.fileupload').fileupload({
                     url: '/services/content',
                     paramName: 'file',
                     dataType: 'json',
+                    formData: {directive: "STORE_AND_PROCESS"},
```

Comment:
The jQuery File Upload Plugin was not sending the headers defined in $.ajaxSetup.  This resulted in the directive being null on the Content Endpoint, and was overlooked since STORE_AND_PROCESS is the default when the directive is null.  This change fixes this problem.